agent/trufflehog_agent.py

```diff
@@ -70,24 +70,22 @@ def process(self, message: m.Message) -> None:
         """
         logger.info("Processing input and Starting trufflehog.")
 
-        if message.selector == "v3.asset.link":
+        if message.selector.startswith("v3.asset.link"):
             link = message.data.get("url", "")
             link_type = process_input.get_link_type(link)
             if link_type is None:
                 return
             cmd_output = self.run_scanner(link_type, link)
-        elif message.selector == "v3.asset.file":
+        elif message.selector.startswith("v3.asset.file"):
             cmd_output = process_input.process_file(message.data.get("content", b""))
-        elif message.selector == "v3.capture.logs":
+        elif message.selector.startswith("v3.capture.logs"):
             content = message.data.get("message", "")
             cmd_output = process_input.process_file(content.encode("utf-8"))
-        elif message.selector == "v3.capture.request_response":
+        elif message.selector.startswith("v3.capture.request_response"):
```

Comment:
This code is not sound because you can have a cmd_output not set. Please declare the variable at the top and set it to None.

Reply:
I missed that, Added. [76ebc4d](https://github.com/Ostorlab/agent_trufflehog/pull/7/commits/76ebc4db77cb27d4d473dd1deaa0131dd0b9d3ed)